tests/python/unittest/test_viz.py

```diff
@@ -55,6 +56,7 @@ def test_plot_network():
     net = mx.sym.SoftmaxOutput(data=net, name='out')
     with warnings.catch_warnings(record=True) as w:
         digraph = mx.viz.plot_network(net, shape={'data': (100, 200)},
+                                      dtype={'data': np.float32},
```

Comment:
should we be testing for dtypes other than `np.float32`?

Reply:
It does not really matter - this test just tries to catch errors during preparation of the picture, and for that the exact type used does not make any difference.